cfme/cloud/instance/__init__.py

```diff
@@ -128,7 +128,7 @@ def is_displayed(self):
             relationship_provider_name = relationships.get_text_of('Cloud Provider')
         except (NameError, NoSuchElementException):
             logger.warning('No "Cloud Provider" Relationship, assume instance view not displayed')
-            return False
+            raise NotImplementedError("This view has no unique markers for is_displayed check")
```

Comment:
@nachandr can you please change it to `is_displayed = displayed_not_implemented`... please refer https://github.com/ManageIQ/integration_tests/pull/7989